### PR TITLE
Added combobox to the main view for displaying unique labels individually

### PR DESCRIPTION
**Pull request by hkaraoguz**
By using a combobox, unique labels can be seen and displayed easily.

### CODE REVIEW

File: combobox.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import sys
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
```
Comment (tzutalin) [inline combobox.py]: Could you avoid importing all(`*`)? Also, supporting PyQt4 is a virtue

Comment (hkaraoguz) [reply combobox.py]: I have fixed the imports and added PyQt4 support based on the implementation in labelImg.py script. I don't have PyQt4 on my environment so I couldn't test the support myself.

Comment (hkaraoguz) [reply combobox.py]: @tzutalin Any chance that you can review this again? Thank you

**Comment by tzutalin** [comment]
Merge it

**Pull request merged**